static/src/javascripts/projects/commercial/modules/messenger/get-stylesheet.js

```diff
@@ -1,42 +0,0 @@
-const getStyles = (specs, styleSheets) => {
-	if (!specs || typeof specs.selector !== 'string') {
-		return null;
-	}
-
-	const result = [];
-	for (let i = 0; i < styleSheets.length; i += 1) {
-		const sheet = styleSheets[i];
-		const ownerNode = sheet.ownerNode;
-
-		if (
-			ownerNode &&
-			ownerNode.matches &&
-			ownerNode.matches(specs.selector)
-		) {
-			if (ownerNode.tagName === 'STYLE') {
-				result.push(ownerNode.textContent);
-			} else {
-				result.push(
-					Array.prototype.reduce.call(
-						sheet.cssRules || [],
-						(res, input) => res + input.cssText,
-						'',
-					),
-				);
-			}
-		}
```

Comment:
I don't understand what this case was handling 🤔 Why is it no longer needed?

Reply:
The code in the else didn't actually work because the only reason it wouldn't be a style element with inline css would be if it was a link element, and trying to access `sheet.cssRules` on a link element with a different domain (like `assets.guim.co.uk`) results in a CORS error so doesn't work any way.

`get-stylesheet` is also only used to get the fonts anyway, which are inlined.

Reply:
Ah ok gotcha. Thanks for the explanation and comment!